### ISSUE TITLE
document how to run Lantern circumvention software in Qubes

### ISSUE THREAD

**Issue by mfc** [QubesOS/qubes-issues#1937]
user issue: https://groups.google.com/d/msgid/qubes-users/51351c19-3850-4944-aa5a-6c7a9b40dad4%40googlegroups.com

1. Download Lantern from https://getlantern.org
2. create debian-8 template for Lantern, `debian-8-lantern`
3. transfer Lantern from AppVM to new `debian-8-lantern` template
4. install Lantern .deb (click on it) in `debian-8-lantern` template
5. install `libappindicator3-1` in template (missing Lantern dependency): `sudo apt-get install libappindicator3-1`
6. create `lantern` appvm based on debian-8-lantern template

--> Lantern software works fine

if you want a lantern-based proxyvm (preferred), currently not sure how to implement successfully. Lantern does not seem to proxy traffic that goes through it from another VM.

**Comment by adrelanos** [comment]
Someone in the Whonix forums just asked how to use latern with
sys-whonix where I gave some DIY / TODO research hints.

https://forums.whonix.org/t/sys-whonix-cant-connect-proxy-vm-use-lantern-proxy-software

**Comment by mfc** [comment]
unman has provided a solution for Lantern, not tested: https://groups.google.com/d/msgid/qubes-users/20160428021206.GA26188%40thirdeyesecurity.org

the much larger issue is users easily enabling obfuscating bridges in `sys-whonix` which I will create an issue for.

**Comment by adrelanos** [comment]
Btw on how to combine Lantern with Whonix, a lot progress has been made.

- https://www.whonix.org/blog/connecting-to-lantern-before-tor
- https://www.whonix.org/wiki/Lantern

> the much larger issue is users easily enabling obfuscating bridges in sys-whonix which I will create an issue for.

https://github.com/QubesOS/qubes-issues/issues/1938

**Comment by ben-grande** [comment]
I don't think this is a topic for the docs, it might be for the forums. Also, few requests to run Lantern compared to any other VPN, disregarding technology merits. @andrewdavidwong close?
